### ISSUE TITLE
Legacy Header: en-header/footer sind in deutsch

### ISSUE THREAD

**Issue by christianTF** [mschlenstedt/Loxberry#661]
LBUpdate heute morgen ausgeführt - darin wird generatelegacytemplate --force durchgeführt und die Templates wurden neu erstellt.
Dennoch enthalten die EN-Versionen von header.html und footer.html deutsche Texte.

![image](https://user-images.githubusercontent.com/14220265/37615508-c829d048-2bad-11e8-9673-4ce7a560369f.png)

![image](https://user-images.githubusercontent.com/14220265/37615601-fcd18f98-2bad-11e8-89f4-4a9ddd3784c0.png)

**Comment by christianTF** [comment]
Language cache problem.... (switching language in one script run). Took some time to find....

**Comment by christianTF** [comment]
Fixed with https://github.com/mschlenstedt/Loxberry/commit/a529f948131fe55b82ca52e35e400ae9d3c4ea91

Clear language cache after generation of each language.

@zmata Could you please verify the correction by updating to Latest Commit. You'll already get your translation file with that update.
I have a minor request for the Czech translation: 
We call "LoxBerry Update" as "LoxBerry Update" in every language as its proper name of the function. This makes support easier and we think that this name is understood in every language.
Therefore, could you please change the function name, now translated as _Aktualizace LoxBerry_, back to _LoxBerry Update_ in every occurance. 
Thanks a lot for your support!!

Christian

**Comment by zmata** [comment]
OK I'll  install Latest Commit  and check it.
LoxBerry Update instead of Aktualizace LoxBerry isn't problem for us. So I'll change it and let you know.
Thanks
Zmata

**Comment by zmata** [comment]
In Latest Commit in case of the English everything is OK. Language of the Squeezelite is EN and Widgets are in EN too.
But in case of the Czech, everything fall down to German. It doesn't mater if I have Czech translation files of the Squeezelite in /opt/loxberry/templates/plugins/squeezelite/lang or not.
I'd expect, that if there is translation, it's in Czech. If there isn't translation, it falls to English.
Zmata

**Comment by christianTF** [comment]
@zmata You are absolutely right - bringing me back to the plugin code ;-)

**Comment by zmata** [comment]
Czech LoxBerry's language file is corrected: https://github.com/zmata/LoxBerryCz

**Comment by christianTF** [comment]
@zmata I think it is done now :-)

https://github.com/christianTF/LoxBerry-Plugin-squeezelite/commit/58e4b95677292cd39458ef83ad60db29a93f1e75
- Fixes fallback language of Squeezelite plugin to English

https://github.com/mschlenstedt/Loxberry/commit/62a38a2ad5d26a2e57a28433764ece38c8b582e7
- Do not send language in query string in the footer buttons (as they would request the fallback language of the main pages).
- Updated language_cs.ini

Could you please do another LoxBerry Update and an update of the plugin too:
Could you install this ZIP: https://github.com/christianTF/LoxBerry-Plugin-squeezelite/archive/dev.zip

There is one issue left that cannot be handled in legacy plugins (that are plugins written for LoxBerry 0.2.x):
If the plugin switches to English (because the plugin does not ship with the configured language), also the left and right panels use the fallback language. Clicking any widget on the left or on the button should bring you back to your configured language.
This issue only applies for legacy plugins. Plugins using the LoxBerry 1.0 template system are not affected.

Sorry for the inconvenience! You now are some kind of "language beta-tester" as we still have little experience yet handling our (own) new language system :-)

**Comment by zmata** [comment]
@christianTF Partly OK.
Problem is with Czech translation.
Here are Squeezelite Czech files: https://github.com/zmata/LoxBerryCz/tree/master/plugins/squeezelite
Try it. I only used the English one and translated few lines (for example LMSADDRESS).
Squeezelite falls down to English in spite of existing of the Czech translation.

I hope I'm good tester for this wonderfull system ;-)

**Comment by christianTF** [comment]
Yes, you _are_ a good tester!

I have removed another part of language detection from Squeezelite that forces to fall back to en.
It is quite difficult to use the new language system in the plugin without breaking it for LoxBerry 0.2.x, and is getting better step by step.

I also included your partly translated files to the plugin.
 
https://github.com/christianTF/LoxBerry-Plugin-squeezelite/archive/dev.zip

**If you already have translated further, make a backup of your plugin translation files before installing!**
